labelbox/schema/iam_integration.py

```diff
@@ -0,0 +1,31 @@
+from labelbox.orm.db_object import DbObject
+from labelbox.orm.model import Field
+
+
+class IAMIntegration(DbObject):
+    """ Represents an IAM integration for delegated access
+
+    Attributes:
+        name (str)
+        updated_at (datetime)
+        created_at (datetime)
+        provider (str)
+        valid (bool)
+        last_valid_at (datetime)
+        is_org_default (boolean)
+
+    """
+
+    def __init__(self, client, data):
+        self.settings = data.pop('settings', {})
```

Comment:
instead of keeping a dict, could we map it to a dataclass based on `__typename`? (Or to Entity - not sure how we represent "helper" gql types like eg `GcpIamIntegrationSettings`?)